android-components/components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -130,14 +130,20 @@
     <string name="mozac_feature_prompts_manage_address">Manage addresses</string>
 
     <!-- Federated Credential Management prompts -->
+    <!--Content description for the Provider favicon in the Select Provider FedCM prompt -->
```

Comment:
You might want to be a little more descriptive and avoid some of the business acronyms for the comments, since a lot of the localizers don't necessarily poke into everything in the app to be able to have the context.

Reply:
I realised I ended up not using this string, as I think there's no need for the Screen Reader to read a view that is purely decorative. (see [Cases that don't require content labels in Google Documentation](https://support.google.com/accessibility/android/answer/7158690?hl=en). 

Thanks for pointing this out!